elastic/kibana
# [Response Ops] [Alerting] Rules and Connectors: Current page in breadcrumbs shows as link

Resolves https://github.com/elastic/kibana/issues/136969

## Summary

The breadcrumbs in Rules and Connectors show the last item (current page) as a link, this pr allows you to conditionally send back the href for the bread crumb.

### Checklist


### To Verify

- Navigate to the Rules and Connectors page
- Click on the different tabs, Rules, Connectors, and Logs
- Verify that The last item in our breadcrumbs should show the current page and be styled differently from the previous / parent pages. It should not be clickable compared to the parent pages, which are clickable (the reason for the difference in styling).
- Click into the rules details and verify the same as above
